### DISCUSSION TITLE
Get text from CTkEntry

### DISCUSSION THREAD

**Discussion by ikeyler** [TomSchimansky/CustomTkinter#788] [Q&A] [answered]
How to get the text from the CTkEntry? I tried get() but nothing happens.
![image](https://user-images.githubusercontent.com/119964453/205984970-d845a745-b33e-4da9-a8fc-4d366628f94e.png)
![image](https://user-images.githubusercontent.com/119964453/205984974-b868cc5c-b259-4221-9be4-7d26f92dbcb6.png)

**Comment by TomSchimansky** [comment, marked as answer]
the `pack` method does not return the widget but None. You have to call `pack` separately.
